prov/verbs/src/fi_verbs.c

```diff
@@ -316,20 +316,17 @@ fi_ibv_send(struct fi_ibv_msg_ep *ep, struct ibv_send_wr *wr, void *context)
 	if (ep->scq) {
 		if (wr->send_flags & IBV_SEND_SIGNALED) {
 			fastlock_acquire(&ep->wre_lock);
-			wre = util_buf_alloc(ep->wre_pool);
+			wre = util_buf_alloc(ep->send_wre_pool.pool);
 			if (OFI_UNLIKELY(!wre)) {
 				fastlock_release(&ep->wre_lock);
 				return -FI_EAGAIN;
 			}
-			memset(wre, 0, sizeof(*wre));
-			dlist_insert_tail(&wre->entry, &ep->wre_list);
```

Comment:
Please update the commit message that explains (what/why) this change.

Reply:
Okay, I think I got why this is being removed here. It seems fi_ibv_wre_buf_free is now handling releasing unreleased buffers. Still it would be nice to have details in the commit message.

Also, util_buf_pool_destroy has the following assertion before the free_hndlr is called:
`assert(buf_region->num_used == 0);`
So we would hit it before any unreleased buffers got released.